src/mdbExtensionController.ts

```diff
@@ -815,6 +816,41 @@ export default class MDBExtensionController implements vscode.Disposable {
     }
   }
 
+  async showSurveyForEstablishedUsers(): Promise<void> {
+    const surveyId = '9viN9wcbsC3zvHyg7';
+
+    const hasBeenShownSurveyAlready =
+      this._storageController.get(StorageVariables.GLOBAL_SURVEY_SHOWN) ===
+      surveyId;
+
+    // Show the overview page when it hasn't been show to the
+    // user yet, and they have saved connections
+    // -> they haven't just started using this extension
+    if (!hasBeenShownSurveyAlready) {
+      if (this._connectionStorage.hasSavedConnections()) {
+        const action = 'Share your thoughts';
+        const text = 'How can we make the MongoDB extension better for you?';
+        const link = 'https://forms.gle/9viN9wcbsC3zvHyg7';
+        const result = await vscode.window.showInformationMessage(
```

Comment:
Is this something we also want to track in telemetry? Like how many folks are shown the survey and dismiss or click? cc @mmarcon